controller/array_action/array_mediator_xiv.py

```diff
@@ -16,6 +16,10 @@
 array_connections_dict = {}
 logger = get_stdout_logger()
 
+LUN_IS_ALREADY_IN_USE_ERROR = "LUN is already in use"
+THE_REQUESTED_MAPPING_IS_NOT_DEFINED_ERROR = "The requested mapping is not defined"
+NO_SPACE_TO_ALLOCATE_TO_THE_VOLUME_ERROR = "No space to allocate to the volume"
```

Comment:
consider shorter names, e.g. `UNDEFINED_MAPPING_ERROR` and `NO_SPACE_(FOR_VOLUME_)?ERROR`

Reply:
done